task4.js

```diff
@@ -0,0 +1,49 @@
+////////////////////////////////////////////////////////////////
+// Working with a car object
+let car = {
+	color: "black",
+}
+car.color = "green"
+car.power = function(enginePower = 0) {
+	console.log(`the engine power is ${enginePower}`)
+}
+//car.power();
+//car.power(10);
+////////////////////////////////////////////////////////////////
+// Working with the warehouse
+function calcNumberOfFruit(nPears, nApples) {
+	return nPears + nApples
+}
+//console.log(calcNumberOfFruit(4, 5));
+////////////////////////////////////////////////////////////////
+// Working with the terminal
+function getName(name) {
+	if (name) {
```

Comment:
Not quite right: the function must compare the passed name to some specific value.